src/main/java/com/epam/cdp/m2/hw2/aggregator/Java7Aggregator.java

```diff
@@ -0,0 +1,84 @@
+package com.epam.cdp.m2.hw2.aggregator;
+
+import java.util.*;
+
+import javafx.util.Pair;
+
+public class Java7Aggregator implements Aggregator {
+
+    @Override
+    public int sum(List<Integer> numbers) {
+        int sum = 0;
+
+        for (int num : numbers) {
+            sum += num;
+        }
+
+        return sum;
+    }
+
+    @Override
+    public List<Pair<String, Long>> getMostFrequentWords(List<String> words, long limit) {
+        List<Pair<String, Long>> result = new ArrayList<>();
+
+        for (String word : words) {
+            Pair<String, Long> pair = findPairByKey(result, word);
+            long frequency = 1L;
+
+            if (pair != null) {
+                frequency = pair.getValue() + 1;
+                result.remove(pair);
+            }
+
+            result.add(new Pair<>(word, frequency));
+        }
+
+        Collections.sort(result, (p1, p2) -> p2.getValue().compareTo(p1.getValue()));
```

Comment:
You can't use Lambda here.

Reply:
yeah, i mean. task says i'm not allowed to use Java 8 packages, not Java 8 syntax. in this case lambda here is just syntax sugar. do you want me to replace it with anonymous class here?

Reply:
yes